_posts/2016/2016-01-15-setup-a-coreos-cluster-just-for-fun-on-centos-and-win.md

```diff
@@ -0,0 +1,262 @@
+On windows there are some more things to observe.
+
+# Windows
+
+## Prerequisits
+
+### 1. Power Shell 3.0
+First you have to install the Windows Management Framework 3.0 including the Windows PowerShell 3.0.
+The standard shell will have some troubles with pathes.
+- Download the latest version, e.g. from this link https://www.microsoft.com/en-us/download/details.aspx?id=34595
+
+###  2. install cygwin
+To get the rsync feature you need to install cygwin on your windows installation.
+- You can download it from here https://cygwin.com/install.html
+
+> *Make sure you activate "rsync" and "SSH" in the installation*
+
+- add "C:\cygwin64\bin" to your windows system variable "PATH"
+
+### 3. [VirtualBox](https://www.virtualbox.org/)
+Virtual Box is needed, download latest version from here https://www.virtualbox.org/wiki/Downloads .
+
+### 4. [Vagrant](https://www.vagrantup.com)
+You also need to install vagrant, get the latest version from here https://www.vagrantup.com/downloads.html .
+
+## Lets get started
+### 1. clone the coreos-vagrant git repo
+
+> *Make sure you use the GitBash for this.*
+
+```
+$ git clone https://github.com/coreos/coreos-vagrant.git
+$ cd coreos-vagrant
+```
+
+### 2. start a cluster
+We need to generate a unique token where etcd can find information about the nodes of the etcd cluster.
+- get dicovery token from https://discovery.etcd.io/new
+- edit file `user-data` replace "https://discovery.etcd.io/NEW_TOKEN" with the url you get above
+```
+etcd2: 
+discovery: https://discovery.etcd.io/NEW_TOKEN
+```
+
+- after replacing the link it should looks like this
+```
+etcd2: 
+discovery: https://discovery.etcd.io/0850c01e29cb6d7dcd5d4e9123311704
+```
+
+### 3. start clusters
+now change to your powershell 3 and execute following commands
+```
+$ cd coreos-vagrant
+$ vagrant up
+```
+If everything works fine you should see three virtualbox windows and your shell should look like this:
+
+{% image 2016_01_coreos_winsetup__01.jpg %}{% endimage %}
+
+### 4. stop clusters
+to stop the clusters execute following commands in your powershell 3
+```
+$ cd coreos-vagrant
+$ vagrant halt
+```
+
+### 5. increase cluster count
+to increase the count of clusters we have to change the file 'config.rb'
+lets try it to set up coreos with 10 clusters just for fun
+
+lets change this
+```
+# Size of the CoreOS cluster created by Vagrant
+$num_instances=3
+```
+
+to this
+```
+# Size of the CoreOS cluster created by Vagrant
+$num_instances=10
+```
+
+{% image 2016_01_coreos_winsetup__02.jpg %}{% endimage %}
+
+now change to your powershell 3 and execute following commands
+```
+$ cd coreos-vagrant
+$ vagrant up
+```
+
+if everything works fine you should see something like this
+
+{% image 2016_01_coreos_winsetup__03.jpg %}{% endimage %}
+
+### 6. remove clusters
+to completely remove the clusters execute following commands in your powershell 3
```

Comment:
Change to: To completely remove the clusters execute the following commands in your PowerShell 3.0: